src/client/app/warehouses/warehouses.controller.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('app.warehouses')
+    .controller('WarehousesController', WarehousesController);
+
+  WarehousesController.$inject = ['$q', 'warehousesService', 'logger'];
+  /* @ngInject */
+  function WarehousesController($q, warehousesService, logger) {
+    var vm = this;
+
+    vm.title = 'Warehouses';
+
+    activate();
+
+    function activate() {
+      getWarehouses().then(function () {
+        logger.info('Warehouse View');
+      });
+    }
+
+    function getWarehouses() {
+      return warehousesService.getWarehouses().then(function (data) {
```

Comment:
move `warehousesService.getWarehouses().` to activate to get rid of this function